Android/Guava/build.cake

```diff
@@ -1,28 +1,27 @@
-
-#load "../../common.cake"
-
 var TARGET = Argument ("t", Argument ("target", "Default"));
 
-var GUAVA_VERSION = "23.2-android";
-var NUGET_VERSION = "23.2.0";
+var GUAVA_NUGET_VERSION = "27.1";
+var GUAVA_FAILUREACCESS_NUGET_VERSION = "1.0.1";
+
+var GUAVA_VERSION = GUAVA_NUGET_VERSION + "-android";
```

Comment:
Just as a note: cake now supports [String Interpolation][1], if you like it more than concatenation and `string.Format`.

[1]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated